### DISCUSSION TITLE
Question regarding "tasks"

### DISCUSSION THREAD

**Discussion by N8Solutions** [akeeba/panopticon#635] [Q&A (Community Support)] [answered]
I couldn't find any information on this so wanted to ask here.
In the "System Configuration", I have Joomla Updates set to install Patch Versions Only. For Extensions, I have it set to "do nothing" but have specified overrides for specific extensions.
Now for an individual site, the settings under "Joomla Update" and "extensions" is set to the Global policy.
Under "Troubleshooting --> Tasks" I see this in the photo.
![image](https://github.com/akeeba/panopticon/assets/12389693/b4c7e812-9168-488d-810f-d5df1f4b50dd)
There are the columns "enabled" and "status". Since I have those Global policies specified in the system configuration and for the site, I'm curious to know what these 2 columns mean for the site.

Thanks in advance for your time!

**Comment by nikosdion** [comment, marked as answer]
When a Joomla! update is found and it needs to be installed according to the settings you have provided in all the various places and their combined effective state, the site update director task sets up and enables the `joomlaupdate` task to execute the site update.

When one or more extension updates are found and they need to be installed according to the settings you have provided in all the various places and their combined effective state, the extensions update director task sets up and enables the `extensionsupdate` task to execute the extensions update.

Since there are is neither a Joomla! core update, nor an extensions update taking place these two tasks are disabled so they don't waste any resources.
